asreview/entry_points/simulate.py

```diff
@@ -171,7 +171,7 @@ def _simulate_parser(prog="simulate", description=DESCRIPTION_SIMULATE):
              f"Default {DEFAULT_N_INSTANCES}.")
     parser.add_argument(
         "--n_queries",
-        type=int,
+        type=str,
         default=None,
         help="The number of queries. By default, the program "
```

Comment:
Please add this option to the help. Don't forget to update this manually https://asreview.readthedocs.io/en/latest/API/cli.html#cmdoption-asreview-simulate-n-queries.